pkg/service/manipulator.go

```diff
@@ -58,6 +64,7 @@ func (m *manipulator) Process(spec ProcessSpec) ([]byte, error) {
 	var err error
 	t := time.Now()
 	data, f, err := m.processor.Decode(spec.ImageData)
+	orientation, _ := native.GetOrientation(bytes.NewReader(spec.ImageData))
```

Comment:
this is not needed unless auto=compress parameter is being sent, so i want to call this inside the `if params[auto] == compress {}` instead. but the processed image wouldn't have this information. what if we fix the orientation first before processing everything else?

Reply:
absolutely. i moved the line to the if params[auto] == compress {} block.
What do you mean by the processed image wouldn't have this information? We can get the orientation from the raw bytes so it should not become a problem.
I guess doing it this way is better because we are resizing/cropping first before fixing the orientation right?
Wdyt?